ci/dotnet.yml

```diff
@@ -13,13 +13,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Setup .NET Core
+    - name: Setup .NET
```

Comment:
If we were to suggest this change, then we should also bump dotnet-version as well as 3.1.301 is still 'NET Core' branded  and just for consistency if we are nudging it along.

Reply:
That's #723. I'd be fine waiting on this change until that merges.